packages/rum-core/src/domain/view/viewCollection.spec.ts

```diff
@@ -80,13 +80,15 @@ describe('viewCollection', () => {
   ) {
     getReplayStatsSpy = jasmine.createSpy()
     const domMutationObservable = new Observable<void>()
+    const windowOpenObservable = new Observable<void>()
     const locationChangeObservable = new Observable<LocationChange>()
 
     const collectionResult = startViewCollection(
       lifeCycle,
       mockRumConfiguration(partialConfiguration),
       location,
       domMutationObservable,
+      pageOpenObserable,
```

Comment:
Nit: "Obserable"

Reply:
Also, shouldn't this be `windowOpenObservable`, which you create above?

Reply:
Done using @thomas-lebeau update